src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -4747,7 +4752,7 @@ synchronized private Plan loadService(Plan plan, String name, String type, boole
 
     log.info("loading - {} {} {}", name, type, level);
 
-    if (name.equals("i01.controller3")) {
+    if (name.equals("i01.simulator")) {

```

Comment:
This whole conditional should probably be removed, or at least the log statement set to debug instead of info

Reply:
removed